TestCases/compliance-level-3/0065-feel-disjunction/0065-feel-disjunction.dmn

```diff
@@ -0,0 +1,176 @@
+        <question>Result of FEEL expression 'true or false'?</question>
+        <allowedAnswers>true (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="decision002"/>
+        <literalExpression>
+            <text>true or false</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision003_a" id="_decision003_a">
+        <description>Tests FEEL expression: 'true or null' and expects result: 'true (boolean)'</description>
+        <question>Result of FEEL expression 'true or null'?</question>
+        <allowedAnswers>true (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="decision003_a"/>
+        <literalExpression>
+            <text>true or null</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision003_b" id="_decision003_b">
+        <description>Tests FEEL expression: 'true or 123' and expects result: 'true (boolean)'</description>
+        <question>Result of FEEL expression 'true or 123'?</question>
+        <allowedAnswers>true (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="decision003_b"/>
+        <literalExpression>
+            <text>true or 123</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision003_c" id="_decision003_c">
+        <description>Tests FEEL expression: 'true or "true"' and expects result: 'true (boolean)'</description>
+        <question>Result of FEEL expression 'true or "true"'?</question>
+        <allowedAnswers>true (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="decision003_c"/>
+        <literalExpression>
+            <text>true or "true"</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision004" id="_decision004">
+        <description>Tests FEEL expression: 'false or true' and expects result: 'true (boolean)'</description>
+        <question>Result of FEEL expression 'false or true'?</question>
+        <allowedAnswers>true (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="decision004"/>
+        <literalExpression>
+            <text>false or true</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision005" id="_decision005">
+        <description>Tests FEEL expression: 'false or false' and expects result: 'false (boolean)'</description>
+        <question>Result of FEEL expression 'false or false'?</question>
+        <allowedAnswers>false (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="decision005"/>
+        <literalExpression>
+            <text>false or false</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision006_a" id="_decision006_a">
+        <description>Tests FEEL expression: 'false or null' and expects result: 'false (boolean)'</description>
+        <question>Result of FEEL expression 'false or null'?</question>
+        <allowedAnswers>false (boolean)</allowedAnswers>
```

Comment:
from `false (boolean)`
to `null (boolean)`

Reply:
@tarilabs amended.